eppo/src/androidTest/java/cloud/eppo/android/EppoClientTest.java

```diff
@@ -262,12 +258,14 @@ private void runTestCases() {
     @Test
     public void testCachedAssignments() {
         try {
+            // First initialize successfully
             initClient(TEST_HOST, false, true, false); // ensure cache is populated
 
-            // wait for a bit since file is loaded asynchronously
+            // wait for a bit since cache file is loaded asynchronously
             System.out.println("Sleeping for a bit to wait for cache population to complete");
-            Thread.sleep(1000);
+            Thread.sleep(2000);
```

Comment:
We do still need this sleep though, for asynchronous cache repopulation, but I'm doubling it given how unimpressed I am with GitHub action performance.